README.md

```diff
@@ -1,10 +1,15 @@
 # E-Lesson (functional DDD showcase)
 
-## Exercise #2: Workflows, simple types
-
-1. Finish implementing...
-- the `StartLesson` workflow
-- input & output types (use also `base-types-kt/refined-types` lib)
-- cucumber Given and When steps
-    
-...so that the tests pass.
\ No newline at end of file
+## Exercise #6: Errors part 1
+
+This time we will explore errors. Let's try to implement error handling using exceptions.
+
+### Exercise description
+
+- Go to [CurrentLesson.kt](src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/CurrentLesson.kt)
```

Comment:
whenever I click this link in Intelij, it takes me [here](https://github.com/krzykrucz/e-lesson/blob/master/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/CurrentLesson.kt) not sure if it's expected